storage/innobase/xtrabackup/doc/source/xtrabackup_bin/throttling_backups.rst

```diff
@@ -3,14 +3,17 @@
 Throttling Backups
 ==================
 
-Although xtrabackup does not block your database's operation, any backup can add load to the system being backed up. On systems that do not have much spare I/O capacity, it might be helpful to throttle the rate at which xtrabackup reads and writes data. You can do this with the :option:`--throttle` option, this option limits the number of I/O operations per second in 1 MB units. 
+Although xtrabackup does not block your database's operation, any backup can add
+load to the system being backed up. On systems that do not have much spare I/O
+capacity, it might be helpful to throttle the rate at which xtrabackup reads and
+writes data. You can do this with the :option:`--throttle` option.
```

Comment:
This page does not mention the chunk size anymore. Is it intentional change? We could mention here that chunk size is 10Mb.

Reply:
The chunk size is mentioned in the description of the --throttle option.

Reply:
Ok. Maybe it is enough.